test/integration/host-mode-port-filter.bats

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bats
```

Comment:
Could we call this `port-filter.bats`? I think this would be the right home for other integration tests such as port filtering in bridge mode.